intranet/templates/signage/pages/bus.html

```diff
@@ -81,6 +81,13 @@
         margin-right: 6px;
     }
 
+    .logo {
+        position:absolute;
```

Comment:
I'm _really_ nitpicking but convention dictates that there should be a space after the colon for each of these lines.